### PR TITLE
171 tanzu service mesh workload cluster sizing

### PR DESCRIPTION
**Pull request by itgrl-bex**
Added small note to the Tanzu Service Mesh section regarding workload cluster sizing with link to the TSM requirements doc.

This PR addresses Issue 171

Signed by: Rebecca Robinson <rerobinson@vmware.com>

### CODE REVIEW

**Review by manishjha86 — APPROVED** [review]

File: src/deployment-guides/tko-saas-services.md [hunk 1]
```diff
@@ -257,8 +257,8 @@ You can onboard Tanzu Service Mesh with or without Tanzu Mission Control.
 * (Conditionally required) An active Tanzu Mission Control subscription.  Required only if you intend to onboard a cluster to Tanzu Service Mesh via Tanzu Mission Control.
 	* Ensure that the cluster to be onboarded to Tanzu Service Mesh is attached to Tanzu Mission Control.
 * An active Tanzu Service Mesh subscription.
-* Workload cluster resource requirements:
-    - **Nodes**: At least 3 worker nodes, each with at least 3,000 milliCPU (3 CPUs) of allocatable CPU and 6 GB of allocatable memory.
+* Workload cluster resource requirements: *Please see [TSM requirements](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/tanzu-service-mesh-environment-requirements-and-supported-platforms/GUID-D0B939BE-474E-4075-9A65-3D72B5B9F237.html#:~:text=Tanzu%20Service%20Mesh%20requires%20a,any%20node%20on%20the%20cluster.)*
```
Comment (srinivasme21) [inline src/deployment-guides/tko-saas-services.md:260]: ...: For workload cluster resource requirements, see [Tanzu Service Mesh Environment Requirements and Supported Platforms] (https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/tanzu-service-mesh-environment-requirements-and-supported-platforms/GUID-D0B939BE-474E-4075-9A65-3D72B5B9F237.html)

File: src/deployment-guides/tko-saas-services.md [hunk 2]
```diff
@@ -287,6 +287,14 @@ Note:
 
    ![Namespace Exclusion](img/tko-saas-services/tko-deploy-saas20.jpg)
 
+   - If using user managed packages such as cert-manager, contour, prometheus, and grafana, you will want to exclude those namespaces.
```
Comment (srinivasme21) [inline src/deployment-guides/tko-saas-services.md:290]: If you install user-managed packages such as Cert Manager, Contour, Prometheus, and Grafana, exclude the related namespaces for the user-managed packages:

File: src/deployment-guides/tko-saas-services.md [hunk 3]
```diff
@@ -287,6 +287,14 @@ Note:
...
+     - cert-manager
+     - tanzu-system-monitoring
+     - tanzu-system-dashboards
+     - tanzu-system-ingress
+
+   ![Namespace Exclusion](img/tko-saas-services/tko-deploy-saas23.png)
+
 5. Click Confirm. The Tanzu Mission Control adapter for Tanzu Observability appears in an unhealthy state for a few minutes because the required objects are being created on the cluster.
 
 6. On the target cluster, you will see a new namespace vmware-system-tsm and required objects being created. When all the required Kubernetes objects are created and are in a running state, status of the Tanzu Mission Control adapter changes to OK.
```
Comment (srinivasme21) [inline src/deployment-guides/tko-saas-services.md:300]: Does not need to be separate step. This can be a paragraph under the step 5.

**Review by srinivasme21 — CHANGES_REQUESTED** [review]
Change the screen capture to light background. 

Please update with the comments.

**Review by srinivasme21 — APPROVED** [review]
LGTM

**Pull request merged**